shared-helpers/src/locales/es.json

```diff
@@ -624,7 +624,7 @@
   "nav.browseProperties": "Buscar Propiedades",
   "nav.getFeedback": "Nos encantaría recibir sus comentarios",
   "nav.listings": "Listados",
-  "nav.browseAllListings": "Buscar listados",
+  "nav.viewListings": "Ver listados",
```

Comment:
All the Help Center content references `View Listings` rather than `Browse Listings`, so changed this in the nav to match. It's in the older Figma content as the latter and the new Figma content as the former.